troubadix/plugins/script_calls_empty_values.py

```diff
@@ -57,12 +58,16 @@ def check_content(
                 plugin=self.name,
             )
 
-        matches = _get_special_script_tag_pattern(
-            name=r"(?!add_preferences).*", value=""
-        ).finditer(file_content)
-        for match in matches:
-            yield LinterError(
-                f"{match.group(0)} does not contain a value",
-                file=nasl_file,
-                plugin=self.name,
-            )
+        for call in SpecialScriptTag:
```

Comment:
AFAICT the `SpecialScriptTag` Enum contains various script_tags like e.g. the following:

```
script_name("foo");
```

which don't need to be checked by this plugin at all because they don't have a `value:` parameter.

Unfortunately i don't have a full list of functions which needs to be checked but we might want to ask the scanner team for such one.

Reply:
I will move this into an improvement ticket because the original issue seems to be already solved.